src/Files.App/ServicesImplementation/DateTimeFormatter/AbstractDateTimeFormatter.cs

```diff
@@ -8,7 +8,7 @@ namespace Files.App.ServicesImplementation.DateTimeFormatter
 {
 	internal abstract class AbstractDateTimeFormatter : IDateTimeFormatter
 	{
-		private static readonly CultureInfo cultureInfo = new(ApplicationLanguages.PrimaryLanguageOverride);
+		private static readonly CultureInfo cultureInfo = new(ApplicationLanguages.Languages[0]);
```

Comment:
This doesn't seem right, does it follow the language override as well?

Reply:
Yes.
https://learn.microsoft.com/uwp/api/windows.globalization.applicationlanguages.primarylanguageoverride

> When you set the PrimaryLanguageOverride, this is immediately reflected in the ApplicationLanguages.Languages property.

I have also confirmed that selecting another language instead of the default works fine.